FindOpenBLAS.cmake

```diff
@@ -4,13 +4,15 @@ if( "ilp64" IN_LIST OpenBLAS_FIND_COMPONENTS AND "lp64" IN_LIST OpenBLAS_FIND_CO
 endif()
 
 if( OpenBLAS_PREFERS_STATIC )
-  set( OpenBLAS_LIBRARY_NAME "libopenblas.a" )
+  set( OpenBLAS_LP64_LIBRARY_NAME  "libopenblas.a"    )
+  set( OpenBLAS_ILP64_LIBRARY_NAME "libopenblas_64.a" )
```

Comment:
Digging through their release history, it looks like this naming for ILP64 only exists in their `CMake` build (since 2019), not in the `Makefile` based build.